README.md

```diff
@@ -66,8 +66,35 @@ This message type has been designed to fit within another element on the page. I
 </div>
 ```
 
+Notice message types are styled similarly to the alert type messages (both for a fit under a header, or inside another element).
+The differences are that they do not have an icon, they don't require highlighted content, and don't support additional content.
+
+```html
+<div class="o-message o-message--alert-inner o-message--success" data-o-component="o-message">
+	<div class="o-message__container">
+		<div class="o-message__content">
+			<p class="o-message__content--main">The quick brown fox jumped over the lazy dogs!</p>
+
+			<div class="o-message__actions">
+				<a href="#" class="o-message__action--primary">Button</a>
```

Comment:
Oops missed this before. Primary/secondary classes are also BEM elements rather than modifiers.